README.md

```diff
@@ -172,6 +172,20 @@ You can use the option with `-q` to use the output directly.
 $ build/rv32emu -d - out.elf -q | jq .x10
 ```
 
+## Tools
+```shell
+make tool
+```
+### rv_histogram
+This is a static analysis tool for target program.
+
+```shell
+build/rv_histogram [-ar] [target_program_path]
+```
+It has two optional options:
+* `-a`: output the analysis in ascending order(default is descending order)
+* `-r`: output usage of registers(default is usage of instructions)
```

Comment:
Give some reference output as [rv8](https://github.com/michaeljclark/rv8) does.

You may create image file(s) in `docs` directory.